backend/contact/api/config/config.go

```diff
@@ -42,17 +42,16 @@ func (config *Config) setDefaults() {
 
 	// Set default App configuration
 	config.SetDefault("APP_ADDR", ":5000")
-	config.SetDefault("APP_ENV", "local")
```

Comment:
@kachar was there a reason for this setting? I saw it's not used and deleted it

Reply:
`APP_ENV` is just a way to differentiate the running environment. It's not used anywhere as far as i recall.